prometheus.php

```diff
@@ -32,12 +37,13 @@
 
 	add_filter( 'vip_prometheus_collectors', function ( array $collectors, string $hook ): array {
 		$to_init = [
-			'cache'   => Cache_Collector::class,
-			'apcu'    => APCu_Collector::class,
-			'opcache' => OpCache_Collector::class,
-			'login'   => Login_Stats_Collector::class,
-			'error'   => Error_Stats_Collector::class,
-			'post'    => Post_Stats_Collector::class,
+			'cache'                           => Cache_Collector::class,
+			'apcu'                            => APCu_Collector::class,
+			'opcache'                         => OpCache_Collector::class,
+			'login'                           => Login_Stats_Collector::class,
+			'error'                           => Error_Stats_Collector::class,
+			'post'                            => Post_Stats_Collector::class,
+			'potential_multi_dataset_queries' => Potential_Multi_Dataset_Queries_Collector::class,
```

Comment:
Should this only be added when `VIP_POTENTIAL_MULTI_DATASET_QUERIES_COLLECTOR_ENABLED` is set?

Reply:
It's fine, if you expand the diff down you'll see there's a check with `class_exists`